server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/reactivestreams/InputStreamPublisher.scala

```diff
@@ -46,7 +53,7 @@ class InputStreamPublisher[F[_]](range: InputStreamRange, chunkSize: Int)(implic
           case _                                        => chunkSize
         }
 
-        val _ = monad
+        runAsync(monad
```

Comment:
**suggestion:** I think we still should add a comment that in case of `Id` this isn't really async and we are aware that this case violates reactive streams.